.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+`<changelog>Replace text with suggested changelog entry for this PR.</changelog>`
+
+<!-- Examples: -->
+<!-- `<changelog>Enabled newly loaded labels to appear faster on the screen.</changelog>` -->
+<!-- `<changelog>Added an option to set the minimum and maximum pitch of a `Map`.</changelog>`-->
+<!-- `<changelog>Introduced `in` expression for testing whether an item exists in an array or a substring exists in a string.</changelog>`
+<!-- `<changelog>Significantly improved offline pack download performance by marking resources as used in batches.</changelog>`
+<!-- `<changelog>Fixed a bug where non-overlapping symbols would be sorted incorrectly with `MGLSymbolStyleLayer.symbolSortKey`.</changelog>` -->
```

Comment:
Should we have the PR number here? (even if fake)

Reply:
Nope, because the changelog-draft script pulls the PR number automatically. So there’s no need to add it to the PR description